docs/core-concepts/working-with-templates/data-objects.md

```diff
@@ -0,0 +1,47 @@
+# Data Available in Templates
+
+Apostrophe makes different data available for you in templates depending on the context. Most of this is covered in other sections as well, but this page serves to collect that information in one place, as well as highlight options not covered elsewhere.
+
+::: tip
+Anything attached to the `req.data` object becomes visible as the `data` object in Nunjucks templates when rendering pages and other templates. Before a page is rendered, you can add properties onto `req.data` to make any new information that you need available in a template.
+:::
```

Comment:
This is fine. You might mention that an `apostrophe-pages:beforeSend` promise event handler is a popular place to do that, or not.